pkg/secretsencrypt/controller.go

```diff
@@ -38,7 +39,7 @@ type handler struct {
 	ctx           context.Context
 	controlConfig *config.Control
 	nodes         coreclient.NodeController
-	secrets       coreclient.SecretController
+	cclient       typev1.CoreV1Interface
```

Comment:
Other places in the code we just call this `core`, I think.

Reply:
You could probably just make this a `kubernetes.Interface` and keep a reference to `k8s` that is passed in to Register, and call `h.k8s.CoreV1().Secrets()`